src/components/CalendarButton/CalendarButton.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import { findDOMNode } from 'react-dom';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import { START_DATE } from 'react-dates/constants';
+
+import { RangePickerController } from '../Calendar';
+import Button from '../Button';
+
+class CalendarButton extends Component {
+  state = { startDate: null, endDate: null, focusedInput: null };
+  buttonRef = null;
+
+  _onDatesChange = ({ startDate, endDate }) => {
```

Comment:
We generally don't use the underscore prefix for any methods. And we've been moving to naming using the following pattern for handlers in class components: `handle{Action}{Object}`. So here that would be `handleChangeDates`.

Reply:
Sure. Changed all occurrences.